backend/app/views/spree/admin/promotions/index.html.erb

```diff
@@ -16,33 +16,40 @@
   <div data-hook="admin_promotions_index_search">
     <%= search_form_for [:admin, @search] do |f| %>
       <div class="row">
-        <div class="col-3">
+        <div class="col-4">
           <div class="field">
             <%= label_tag :q_name_cont, Spree::Promotion.human_attribute_name(:name) %>
             <%= f.text_field :name_cont, tabindex: 1 %>
           </div>
         </div>
 
-        <div class="col-3">
+        <div class="col-2">
           <div class="field">
             <%= label_tag :q_codes_value_cont, Spree::Promotion.human_attribute_name(:code) %>
             <%= f.text_field :codes_value_cont, tabindex: 1 %>
           </div>
         </div>
 
-        <div class="col-3">
+        <div class="col-2">
           <div class="field">
             <%= label_tag :q_path_cont, Spree::Promotion.human_attribute_name(:path) %>
             <%= f.text_field :path_cont, tabindex: 1 %>
           </div>
         </div>
 
-        <div class="col-3">
+        <div class="col-2">
           <div class="field">
             <%= label_tag :q_promotion_category_id_eq, Spree::PromotionCategory.model_name.human %><br>
             <%= f.collection_select(:promotion_category_id_eq, @promotion_categories, :id, :name, { include_blank: t('spree.match_choices.all') }, { class: 'custom-select fullwidth' }) %>
           </div>
         </div>
+
+        <div class="col-2">
+          <div class="field">
+            <%= label_tag :active, 'Show Only Active' %><br>
+            <%= f.check_box :active, label: false, as: :boolean, inline_label: "#{ t 'spree.active'}", checked_value: true %>
```

Comment:
Do we need the `inline_label` option here?

Reply:
And even if we do, we probably just want to pass in `t('spree.active')` rather than `"#{t 'spree.active'}"` since they should produce the same value.

Reply:
It is not needed. I stripped it out.